fbgemm_gpu/src/cumem_utils.cu

```diff
@@ -357,10 +387,18 @@ Tensor uvm_to_cpu_clone(Tensor t) {
   return cpu_clone;
 }
 
-#ifndef __HIP_PLATFORM_HCC__
-// FIXME: some advanced "cudaMemAdvise" flags are not supported by HIP.
 FBGEMM_GPU_ENUM_GLOGAL(uvm)
-
+#ifdef __HIP_PLATFORM_HCC__
+// FIXME: some advanced "cudaMemAdvise" flags are not supported by HIP.
+FBGEMM_GPU_ENUM_REGISTER_START(uvm, hipMemoryAdvise){
```

Comment:
1. Is this `FIXME` still needed? It looks that all the flags are supported by HIP (at least listed).
2. Can't we rely on hipification script? It looks that the only difference is `hipMemoryAdvise` vs `cudaMemoryAdvise`.

Reply:
Thank you for your suggestions, I'm removing the branch as well as the `FIXME` comment.